accounts/tests.py

```diff
@@ -942,31 +944,31 @@ def test_fix_email_issues_with_both_users_email_change(self):
         self.assertRedirects(resp, reverse('messages'))
 
     def test_replace_when_sending_email(self):
+        return True  # TODO: implement new way of testing this
```

Comment:
What should we do with this? If we want to skip it we can use the `skipIf` decorator as well.

Reply:
Fixed here https://github.com/MTG/freesound/pull/1172/commits/40c503d742541d85fa047798b76b79a3fbbe3b2d